src/api/main/UserController.js

```diff
@@ -257,6 +261,17 @@ export class UserController implements IUserController {
 			}
 		})
 	}
+
+	loadWhitelabelConfig(): Promise<?WhitelabelConfig> {
+
+		// The model allows for multiple domainInfos to have whitelabel configs
+		// but in reality on the server only a single custom configuration is allowed
+		// therefore the result of the filtering all domainInfos with no whitelabelConfig
+		// can only be an array of length 0 or 1
+		return this.loadCustomerInfo()
+		           .then(customerInfo => customerInfo.domainInfos.map(domainInfo => domainInfo.whitelabelConfig).filter(Boolean)[0] || null)
```

Comment:
`.filter(Boolean)[0] || null)` sends a chill down my spine.

why do you need `|| null` in the first place?